src/react-components/avatar-preview.js

```diff
@@ -63,6 +63,26 @@ function fitBoxInFrustum(camera, box, center, margin = DEFAULT_MARGIN) {
   camera.lookAt(center);
 }
 
+function getThemeBackground() {
+  const currentTheme = APP.store.state.preferences.theme;
+  const themes = window.APP_CONFIG?.theme.themes;
+  const defaultColor = 0xeaeaea;
+  if (currentTheme === "hubs-default") {
```

Comment:
If `currentTheme` is "hubs-default", shouldn't it just take the value from the associated theme structure?